### PR TITLE
upd(bat): `0.21.0` -> `0.22.1`

### PR DESCRIPTION
**Pull request by tranzystorek-io**

### CODE REVIEW

**Comment by Henryws** [comment]
Any way to get this working on Debian?

**Comment by tranzystorek-io** [comment]
According to https://github.com/rust-lang/git2-rs/issues/838, debian has much too old version of rust (1.47 on stable afaics)

File: packages/bat/bat.pacscript [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-hash="3dff1e52d577d0a105f4afe3fe7722a4a2b8bb2eb3e7a6a5284ac7add586a3ee"
+hash="25e45debf7c86794281d63a51564feefa96fdfdf575381e3adc5c06653ecaeca"
 maintainer="Orphaned"
-prepare() {
-        true
-}
 
 build() {
-        cargo build --release
+    RUSTONIG_SYSTEM_LIBONIG=1 cargo build -j"$(nproc)" --release --locked
```
Comment (Henryws) [inline packages/bat/bat.pacscript:12]: What's this do?

Comment (tranzystorek-io) [reply packages/bat/bat.pacscript:12]: `RUSTONIG_SYSTEM_LIBONIG` forces linking to the system libonig library, `--locked` makes cargo use dependency versions as set in the `Cargo.toml file`, this leads to better reproducible builds (with regard to upstream release)

Comment (Henryws) [reply packages/bat/bat.pacscript:12]: What's libonig?

Comment (tranzystorek-io) [reply packages/bat/bat.pacscript:12]: The oniguruma regex library that is used as a dependency by bat

**Review by Henryws — APPROVED** [review]

**Pull request merged**